### DISCUSSION TITLE
Multiple clickOutside `.bubble` conflict with each other

### DISCUSSION THREAD

**Discussion by mrleblanc101** [vueuse/vueuse#3823] [Q&A]
Let's say I have a button that open a dropdown.
If I clickOutside, it close the dropdown, good.
![2024-02-25 20 17 10](https://github.com/vueuse/vueuse/assets/17392251/d29e8385-2ad9-4049-85c2-58493384695b)

If I click the button to toggle the open state, both clickOutside and the toggle function gets called, so the dropdown close and re-open, bad.
![2024-02-25 20 17 41](https://github.com/vueuse/vueuse/assets/17392251/2dfea7a1-39cc-4787-be88-42faa9c25ec9)

That's where the `.buddle` and `@click.stop` comes into play.
But if you have multiple dropdown with `.bubble` and `@click.stop`, they cancel each other so when you open a second dropdown, the first dropdown will stay open because of the `.stop`.
![2024-02-25 20 21 20](https://github.com/vueuse/vueuse/assets/17392251/5f62757a-0ed3-4a9f-b239-07debdeaa1f8)

How do I solve this ?

**Comment by erkage** [comment]
Hey,

I solved this with the following code:
```
const searchOutsideHandler = [
  (e) => {
    if (!e.target.classList.value.includes('search'))
      searchOpened.value = false
  },
]
```
But with this it seems that I have to create a new function for every onClickOutside element. At least I haven't find the solution for that yet.
